superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 David Bensoussan, Synapticon GmbH
+# Copyright (c) 2017 Open Source Robotics Foundation, Inc.
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal  in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+
+
+import xmltodict
+import hashlib
+import tarfile
+import glob
+import yaml
+import sys
+import re
+
+from superflore.utils import get_license
+from superflore.utils import UnresolvedDependency
+from superflore.utils import NoPkgXml
+from superflore.utils import resolve_dep
+from termcolor import colored
+
+
+if sys.version_info[0] == 2:
+    import requests
+    import urllib
+
+    def get_http(url):
+        return requests.get(url).text
+else:
+    from urllib.request import urlopen
+    import urllib
+
+    def get_http(url):
+        response = urlopen(url)
+        return response.read()
+
+
+base_url = \
```

Comment:
Yup. That's correct. I should remove that. I also should move the download_yamls function to the utils file, so that I don't duplicate the logic.